R/SSplotProfile.R

```diff
@@ -36,9 +36,9 @@
 #' Default=TRUE.
 #' @param col Optional vector of colors for each line.
 #' @param pch Optional vector of plot characters for the points.
-#' @param lty Line total for the likelihood components.
+#' @param lty Line type for the likelihood components.
 #' @param lty.total Line type for the total likelihood.
-#' @param lwd Line width for the likelihood components.
+#' @param lwd Line width for the likelihood components. 
```

Comment:
Use the template here?

Reply:
Good catch - this one I wasn't sure about because there is also a `@param lwd.total` right below it, and I wasn't sure if the template definition was too general to distinguish between the two?

Reply:
I am fine with leaving it as well as the previous one. There are a few more instances of `@param lwd` though throughout the code. Would you mind making sure we shouldn't change any more of them and then resolving this comment?

Reply:
I just took a look - the reason I left them is they all take vectors instead of a single value, which I thought wasn't in line with the template definition. So I think they should be left.

Maybe renaming them something like `lwd_vec` to make them distinct from `lwd` would make sense. I'll open an issue to this effect for us to consider in the future.